src/coreclr/pgosupport.cmake

```diff
@@ -23,7 +23,10 @@ function(add_pgo TargetName)
         else(CLR_CMAKE_HOST_WIN32)
             if(UPPERCASE_CMAKE_BUILD_TYPE STREQUAL RELEASE OR UPPERCASE_CMAKE_BUILD_TYPE STREQUAL RELWITHDEBINFO)
                 target_compile_options(${TargetName} PRIVATE -flto -fprofile-instr-generate)
-                set_property(TARGET ${TargetName} APPEND_STRING PROPERTY LINK_FLAGS " -flto -fuse-ld=gold -fprofile-instr-generate")
+                set_property(TARGET ${TargetName} APPEND_STRING PROPERTY LINK_FLAGS " -flto -fprofile-instr-generate")
```

Comment:
Just to confirm, LLD supports the same PGO linker flags as GOLD, correct?

Reply:
Correct. I have verified that a release build with -pgoinstrument passes and that the compiler was really passed these options.